tgui/packages/tgui/interfaces/Matterforge.tsx

```diff
@@ -40,7 +40,7 @@ export const LoadedMaterials = (props: MaterialData) => {
       buttons={
         <Button
           icon="arrow-up"
-          tooltip="Load Materials From Hand"
+          tooltip="Toggle Recycle State"
```

Comment:
Please, move this to a separate button.

Since this is React, this is a function that expects a JSX element or nothing, you will need to wrap the button elements in a single element. See this example:
https://github.com/Equinox-SS13/equinox-sojourn/blob/75780e749d1b24972db0ef86f5397026eb1e79b9/tgui/packages/tgui/interfaces/AtmosOmniFilter.tsx#L45-L61